src/Sentry/Protocol/Span.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Sentry.Internal.Extensions;
+
+namespace Sentry.Protocol
+{
+    // https://develop.sentry.dev/sdk/event-payloads/span
+    public class Span : ISpan, IJsonSerializable
+    {
+        public SentryId SpanId { get; }
+        public SentryId? ParentSpanId { get; }
+        public SentryId TraceId { get; set; }
+        public DateTimeOffset StartTimestamp { get; set; }
```

Comment:
I can't see why this has a setter, could you please elaborate?

Reply:
Removed